### ISSUE TITLE
[Data catalog] Document remaining tables and views

### ISSUE THREAD

**Issue by jeancochrane** [ccao-data/data-architecture#26]
The initial dbt project we added in https://github.com/ccao-data/data-architecture/pull/23 contains documentation for many of the Athena views that we build on top of our source data, but not all of them. It also does not contain documentation for source tables yet. Add documentation for these tables and views.

**Comment by jeancochrane** [comment]
Views have been added in https://github.com/ccao-data/data-architecture/pull/64, but we're blocked on documentation by #61.

**Comment by jeancochrane** [comment]
Superseded by #113.
